packages/driver/types/remote-state.d.ts

```diff
@@ -0,0 +1,14 @@
+declare namespace Cypress {
```

Comment:
likely the changes here need to be overwritten to satisfy is [comment](https://github.com/cypress-io/cypress/pull/21237/files#r866899041)